qiita_db/test/test_reference.py

```diff
@@ -7,19 +7,22 @@
 # -----------------------------------------------------------------------------
 
 from unittest import TestCase, main
-from os import close, remove
+from os import close, remove, listdir
 from os.path import basename, join
-from tempfile import mkstemp
+from tempfile import mkstemp, mkdtemp
+
+import numpy.testing as npt
 
 from qiita_core.util import qiita_test_checker
-from qiita_db.reference import Reference
+from qiita_db.exceptions import QiitaDBWarning, QiitaDBError
+from qiita_db.reference import Reference, _rename_sortmerna_indexed_db_files
 from qiita_db.util import get_mountpoint, get_count
 
 
 @qiita_test_checker()
 class ReferenceTests(TestCase):
     def setUp(self):
-        self.name = "Fake Greengenes"
+        self.name = "Fake_Greengenes"
```

Comment:
Haha I guess my brain just interprets underscores as spaces these days, I truly honestly did not even see it. Thanks, was just curious :)